Automattic/woocommerce-payments
# Remove unnecessary babel plugins

#### Changes proposed in this Pull Request
Follow-up from #4948 

After dropping IE11 support we don't need those babel plugins anymore. Our current supported browsers don't need any polyfills or runtime transforms for those features.

- Remove all unnecessary babel plugins:
  - `@babel/transform-runtime` – Used to support async/await.
  - `@babel/plugin-proposal-optional-chaining` – Used to support `?.` syntax.
  - `@babel/plugin-proposal-nullish-coalescing-operator` – Used to support `??` operator.
- Remove unused dependency `@babel/plugin-proposal-object-rest-spread`.
- Remove `.babelrc` as we already use `babel.config.js` for the same purpose.

#### Testing instructions
The build process and E2E tests should pass without errors. We can also quickly test it having UPE enabled and following those instructions:
- Build the project running `npm run build`.
- Add a product to the cart and go to **Checkout**
- Should work as expected without errors on the browser's console.

-------------------


**Post merge**
